packages/forms/src/directives/select_control_value_accessor.ts

```diff
@@ -123,7 +131,31 @@ export class SelectControlValueAccessor implements ControlValueAccessor {
 
   private _compareWith: (o1: any, o2: any) => boolean = Object.is;
 
-  constructor(private _renderer: Renderer2, private _elementRef: ElementRef) {}
+  constructor(
+      private _renderer: Renderer2, private _elementRef: ElementRef, private _ngZone: NgZone) {}
+
+  ngAfterViewChecked() {
+    /*
+     * When no option value matches the select value, in order for the first option to be
+     * deselected in IE 11 and Safari (tested in v11-v13), the select element's value property
+     * must be set *after* the option's child text node is appended to the DOM. However, when
+     * rendering options Angular calls the select component's writeValue() method before adding
+     * text nodes to the DOM, so it is necessary to set the value again after checking the view.
+     *
+     * This is also necessary to work around delayed element removal when using the animations
+     * module. Otherwise when a selected option is removed (so no option matches the ngModel
```

Comment:
Minor suggestion to mention that the second problem actually happens in all browsers (since the first paragraph mentions Safari and IE specifically).
```suggestion
     * module (in all browsers). Otherwise when a selected option is removed (so no option matches the ngModel
```

Reply:
Previously it was mentioned at the end of the paragraph, but I moved it to the suggested location to make it more visible.